holoviews/core/operation.py

```diff
@@ -115,19 +125,41 @@ def __call__(self, element, **params):
                                   kdims=element.kdims)
             # Populate the axis layout
             for pos, cell in element.items():
-                processed[pos] = self(cell, **params)
+                if self.p.dynamic and isinstance(cell, HoloMap):
+                    def dynamic_operation(key):
+                        return self(element[pos][key], **params)
+                    processed_cell = self._make_dynamic(cell, dynamic_operation)
+                else:
+                    processed_cell = self(cell, **params)
+                processed[pos] = processed_cell
         elif isinstance(element, HoloMap):
-            mapped_items = [(k, self._process(el, key=k))
-                            for k, el in element.items()]
-            refval = mapped_items[0][1]
-            processed = element.clone(mapped_items,
-                                      group=refval.group,
-                                      label=refval.label)
+            if self.p.dynamic:
+                def dynamic_operation(key):
+                    return self(element[key], **params)
+                processed = self._make_dynamic(element, dynamic_operation)
```

Comment:
Minor suggestion: I might consider using the signature ``self._make_dynamic(element, key, params)`` and moving the nested function into that method (or using a lambda).

This is a very mild preference and if you prefer to leave it as it is, I don't mind either.